ThirdParty.json

```diff
@@ -2,96 +2,119 @@
   {
     "name": "Async++",
     "url": "https://github.com/Amanieu/asyncplusplus",
-    "license": "MIT"
+    "version": "756893feb9e69c098225d5a454a668a2c139e4be",
+    "license": ["MIT"]
   },
   {
     "name": "Catch2",
     "url": "https://github.com/catchorg/Catch2",
-    "license": "Boost Software License 1.0"
+    "version": "2.13.9",
+    "license": ["Boost Software License 1.0"],
+    "notes": "Only used in the tests."
```

Comment:
We can skip these if we are not shipping them. The ThirdParty.json is meant to document dependencies that are included in releases and shipped. If testing code is for CI / Development only and never redistributed by Cesium Native, ie Dev Dependency, we don't have to list it here.

Reply:
I thought it wouldn't hurt for completeness (or for the benefit of someone later comparing the `extern` directory to ThirdParty.json and wondering why they're different), but I'll remove them.